cmd/kube-controller-manager/app/options/options.go

```diff
@@ -104,6 +104,7 @@ func NewCMServer() *CMServer {
 			ClusterSigningCertFile:   "/etc/kubernetes/ca/ca.pem",
 			ClusterSigningKeyFile:    "/etc/kubernetes/ca/ca.key",
 			ReconcilerSyncLoopPeriod: metav1.Duration{Duration: 5 * time.Second},
+			EnableTaintManager:       true,
```

Comment:
Nope - true here, false in the third PR that's in the making. This PR adds a beta feature, and IIUC beta is enabled by default.